src/Presentation/Nop.Web/Areas/Admin/Views/Shared/Components/MultistorePreview/Default.cshtml

```diff
@@ -0,0 +1,22 @@
+﻿@model IList<MultistorePreviewModel>
+
+<div class="btn-group" id="multiStorePreview">
+    <button type="button" class="btn btn-info">
+        <i class="far fa-eye"></i>
+        @T("Admin.Common.Preview")
+    </button>
+    <button type="button" class="btn btn-info dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
+        <span class="caret"></span>
+        <span class="sr-only">&nbsp;</span>
+    </button>
+    <ul class="dropdown-menu" role="menu">
+        @foreach (var multistorePreview in Model)
+        {
+            <li class="dropdown-item">
+                <button type="button" onclick="javascript:OpenWindow('@multistorePreview.Url', 800, 600, true); return false;">
```

Comment:
Would it be okay to change this `Open` to open in a new tab instead of in a new window?

From my experience I have found the popup to be annoying since it is detached from my main window, and (this may be chrome specific) I lose a lot of the browser functionality in these smaller windows
![image](https://github.com/user-attachments/assets/196b60a3-6755-4c5b-9357-f4ba6e9d061f)

```suggestion
                <button type="button" onclick="javascript:window.open('@multiStorePreview.Url', '_blank').focus();">
```